app/forms/applicants/property_value_form.rb

```diff
@@ -0,0 +1,29 @@
+module Applicants
```

Comment:
Other forms are in the module `Applicants` and use `form_for :applicant`. As the object this form applies to is a `LegalAidApplication`, I think this form should be in the module `LegalAidApplications` (also sub-folder `legal_aid_applications`).